### ISSUE TITLE
Exclude sfdx technical files from search

### ISSUE THREAD

**Issue by FabienTaillon** [forcedotcom/salesforcedx-vscode#1075]
**FEATURE REQUEST**
When using vscode search feature, I receive too many results because it includes technical files, specifically what's under .sfdx/typings. It could be cool to have this folder excluded by default on every new project. I don't see any good use case where I would want to get results from these files.

**Describe the solution you'd like**
Automatically exclude the files under .sfdx/typings from vscode search.

**Describe alternatives you've considered**
So far I'm excluding the folder myself in .vscode/settings.json
"search.exclude": {
        "**/.sfdx/typings": true
 }

**Comment by Szandor72** [comment]
Why not add `./sfdx` to .gitignore and tell VS Code to not search in .gitignore? At least that's how we prevent searching in .sfdx folders.

**Comment by lcampos** [comment]
Another alternative, if you're not using git, is to add some exceptions to your search. You can do this by expanding the search options, click the 3 dots button on the bottom-right of the search input field. Then you can add files and folders to exclude or include and those get saved across vscode restarts.

![expanded_search](https://user-images.githubusercontent.com/1041842/53062357-69103a80-3475-11e9-9f2b-a3dcc1750a8a.png)

**Comment by lcampos** [comment]
I'm closing this issue since it is addressed as part of the Salesforce CLI release `7.3.0`. You can see the new search excludes in `.vscode/settings.json` (added below) by running `sfdx force:project:create --projectname dxProject --template standard`.

```
{
  "salesforcedx-vscode-core.push-or-deploy-on-save.enabled": false,
  "search.exclude": {
    "**/node_modules": true,
    "**/bower_components": true,
    "**/.sfdx": true
  }
}
```
cc @ntotten
